gneiss/regression/_mixedlm.py

```diff
@@ -9,8 +9,8 @@
 import pandas as pd
 import statsmodels.formula.api as smf
 from ._model import RegressionModel
-from ._regression import (_intersect_of_table_metadata_tree,
-                          _to_balances)
+from gneiss.util import (_intersect_of_table_metadata_tree,
+                         _to_balances)
 from decimal import Decimal
 from statsmodels.iolib.summary2 import Summary
 
```

Comment:
line 142 ff are not covered by unittest. Could you please add a test that checks that the ValueError is raised